storage/migrations/0003_oasis_3_amend_commission.up.sql

```diff
@@ -0,0 +1,164 @@
+-- Historical and upcoming commission schedule changes
+
+BEGIN;
+
+-- Commissions Amendments
+
+CREATE TABLE IF NOT EXISTS oasis_3.commissions
+(
+  address TEXT PRIMARY KEY NOT NULL REFERENCES oasis_3.accounts(address),
+  schedule JSON
+);
+
+TRUNCATE oasis_3.commissions CASCADE;
+INSERT INTO oasis_3.commissions (address, schedule) VALUES
```

Comment:
We could also roll these into `0001_oasis_3_state_init.up.sql` and also move the table creation in `0000_oasis_3_init.up.sql`.